frontend/src/metabase/components/DateAllOptionsWidget/DateAllOptionsWidget.tsx

```diff
@@ -2,7 +2,7 @@ import { useState } from "react";
 import { t } from "ttag";
 import cx from "classnames";
 
-import DatePicker from "metabase/query_builder/components/filters/pickers/DatePicker/DatePicker";
+import DatePicker from "metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePicker";
```

Comment:
It'd be nice to add named exports here.

```suggestion
import DatePicker from "metabase/admin/datamodel/components/filters/pickers/DatePicker";
```